eng/Publishing.props

```diff
@@ -168,15 +107,15 @@
     This ensures that we don't produce these files in the "Repo source build" builds,
     but we do produce them in both the VMR and the runtime official build.
   -->
-  <PropertyGroup>
-    <_ShouldGenerateProductVersionFiles Condition="'$(DotNetBuildRepo)' == 'true' and '$(DotNetBuildOrchestrator)' == 'true'">true</_ShouldGenerateProductVersionFiles>
-    <_ShouldGenerateProductVersionFiles Condition="'$(DotNetBuildRepo)' != 'true' and '$(DotNetBuildOrchestrator)' != 'true'">true</_ShouldGenerateProductVersionFiles>
+  <PropertyGroup Condition="'$(DotNetBuildOrchestrator)' == 'true'">
```

Comment:
Don't we need to produce the version files in the current msft builds as well?

Reply:
They're produced by the workloads job (set in runtime-official.yml)

Reply:
They were previously also produced when building locally. I assume this isn't the case anymore?

Reply:
Yeah they won't be produced locally now unless you request them